pkgs/top-level/python-packages.nix

```diff
@@ -8556,6 +8556,25 @@ in {
     };
   };
 
+  python-openid = buildPythonPackage rec {
```

Comment:
Because of the size of this file, we are currently moving all new python packages to: `pkgs/development/python-modules/`

```
e.x: pkgs/development/python-modules/python-openid/default.nix
```